webserver/pkgpkr/webservice/recommender_service.py

```diff
@@ -30,13 +48,38 @@ def get_average_monthly_donwloads(self, daily_donwloads_list):
 
     def get_recommendations(self, dependencies):
 
-        # TODO call model
-        # self.model.predict(dependencies)
-
-        recommended_dependencies = []
+        # Connect to our database
+        db = psycopg2.connect(f"host={DB_HOST} user={DB_USER} password={DB_PASSWORD}")
+        cur = db.cursor()
 
+        # Get recommendations from our model
+        recs_per_dep = 2
+        recommended = []
         for dependency in dependencies:
 
+            # Get the corresponding ID
+            cur.execute(f"SELECT id FROM packages WHERE name = '{dependency}'")
+            result = cur.fetchone()
+            if not result:
+                continue
+            packageId = result[0]
+
+            recommendedIds = self.model[self.model.package_a == packageId].sort_values('similarity', ascending=False)['package_b']
+            for identifier in recommendedIds[:recs_per_dep]:
+                cur.execute("SELECT name FROM packages WHERE id = " + str(identifier))
+                result = cur.fetchone()
+                if not result:
+                    continue
+                recommended.append(result[0])
+
+        # Disconnect from the database
+        cur.close()
+        db.close()
```

Comment:
i'm thinking going forward we could pre-read the entire table so we don't have to reconnect and just pull from the local data structure, or probably use Django models. I think this is good for now though